exchanges/bithumb/bithumb_wrapper.go

```diff
@@ -230,17 +230,22 @@ func (b *Bithumb) Run() {
 }
 
 // FetchTradablePairs returns a list of the exchanges tradable pairs
-func (b *Bithumb) FetchTradablePairs(ctx context.Context, asset asset.Item) ([]string, error) {
+func (b *Bithumb) FetchTradablePairs(ctx context.Context, a asset.Item) (currency.Pairs, error) {
 	currencies, err := b.GetTradablePairs(ctx)
 	if err != nil {
 		return nil, err
 	}
 
+	pairs := make([]currency.Pair, len(currencies))
+	var pair currency.Pair
```

Comment:
Placing the pair var in the for loop reads neater for this one and others elsewhere